package.json

```diff
@@ -3,16 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
+    "enzyme": "^2.7.1",
     "eslint": "3.13.1",
     "eslint-plugin-flowtype": "2.29.2",
     "eslint-plugin-react": "6.9.0",
-    "react-scripts": "0.8.4"
+    "react-addons-test-utils": "^15.4.2",
+    "react-scripts": "0.8.4",
+    "react-test-renderer": "*"
   },
```

Comment:
Do you need `jest-cli` as a devDependency? We have that in our Monsanto project, but maybe it isn't needed.